app/src/main/res/layout/include_duckduckgo_browser_webview.xml

```diff
@@ -15,6 +15,7 @@
   -->
 
 <com.duckduckgo.app.browser.DuckDuckGoWebView xmlns:android="http://schemas.android.com/apk/res/android"
+    style="@style/Theme.DuckDuckGo"
```

Comment:
NIT: I would apply the style in the custom view `DuckDuckGoWebView.kt` so we don't have this problem next time we add it to a layout.